companion/src/firmwares/er9x/er9xinterface.cpp

```diff
@@ -91,7 +94,9 @@ inline void applyStickModeToModel(Er9xModelData & model, unsigned int mode)
 
 bool Er9xInterface::loadxml(RadioData &radioData, QDomDocument &doc)
 {
+  #ifdef DEBUG
```

Comment:
Why not qDebug() << ...
If I am not mistaken it won't display anything when in RELEASE mode, no?